metadata.yaml

```diff
@@ -83,19 +109,132 @@ runtimes:
     name: 7.0.0 (Red Hat Fuse)
 - id: nodejs
   name: Node.js
-  description: A JavaScript runtime built on Chrome's V8 JavaScript engine, using an event-driven, non-blocking I/O model for lightweight efficiency.
```

Comment:
Is it ok to modify sections of other runtimes?

Reply:
Ideally, no, but the `yq` tool used to manipulate YAML does this :(

Reply:
I don't know how it;s implemented (need to look into it), but Atomist supposedly can perform these kinds of change without modifying the doc.

Our current tooling makes such changes inevitable